src/styles/SharedStyles.js

```diff
@@ -12,3 +12,11 @@ export const IntroTextStyle = styled.p`
   font-size: 19px;
   line-height: 26px;
 `;
+
+export const AscH1 = styled(Heading)`
+  margin: ${themeSpacing(5)} 0;
+`;
+
+export const AscHeaderSecondary = styled(Heading)`
```

Comment:
Same here, why not  `import {Heading as AscHeading}`  and use the own heading in the application?

Reply:
gedaan